packages/ibm-products/src/components/ConditionBuilder/utils/util.js

```diff
@@ -102,3 +106,43 @@ export const manageTabIndexAndFocus = (currentElement, conditionBuilderRef) => {
     ?.setAttribute('tabindex', '1');
   currentElement?.focus();
 };
+
+const formatDate = (date) => {
+  const day = String(date.getDate()).padStart(2, '0');
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const year = date.getFullYear();
+  return `${day}/${month}/${year}`;
+};
+
+export const valueRenderers = {
```

Comment:
2 minor things. i think we can simplify this to just be something like `getValue` and also for consistency lets stick between either `val` or `value`

Reply:
modified as advised